### PR TITLE
#patch Feature/audit

### PR DESCRIPTION
**Pull request by Greatmar2**
Updates to make use of new auditing setup.
Note that the CurrentIdentityFromXXX classes are placeholders until an actual implementation can be done.
There is also some usage for the placeholder IClaimlessCommandHandler that shall be removed (and ICommandHandler's method signature changed) in a future PR.

### CODE REVIEW

File: EngineBay.Authentication/BasicAuth/CurrentIdentityFromBasicAuth.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public class CurrentIdentityFromBasicAuth : ICurrentIdentity
+    {
+        public CurrentIdentityFromBasicAuth(IHttpContextAccessor httpContextAccessor)
+        {
+            if (httpContextAccessor == null)
+            {
+                throw new ArgumentNullException(nameof(httpContextAccessor));
+            }
+
+            // TODO: Get user from database?
```
Comment (windy-worm) [inline EngineBay.Authentication/BasicAuth/CurrentIdentityFromBasicAuth.cs:15]: I don't see this todo reflected on the Miro board?

Comment (Greatmar2) [reply EngineBay.Authentication/BasicAuth/CurrentIdentityFromBasicAuth.cs:15]: Added

File: EngineBay.Authentication/JwtBearerAuth/CurrentIdentityFromJwt.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            {
+                throw new ArgumentNullException(nameof(httpContextAccessor));
+            }
+
+            // TODO: use the claims principal to get the current identity
+            var name = httpContextAccessor.HttpContext?.User?.Identity?.Name;
+
+            if (name is null)
+            {
+                throw new ArgumentException("Application users not found at ");
```
Comment (windy-worm) [inline EngineBay.Authentication/JwtBearerAuth/CurrentIdentityFromJwt.cs:21]: message does not make sense

File: EngineBay.Authentication/JwtBearerAuth/CurrentIdentityFromJwt.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    public class CurrentIdentityFromJwt : ICurrentIdentity
+    {
+        public CurrentIdentityFromJwt(IHttpContextAccessor httpContextAccessor)
+        {
+            if (httpContextAccessor == null)
+            {
+                throw new ArgumentNullException(nameof(httpContextAccessor));
+            }
+
+            // TODO: use the claims principal to get the current identity
```
Comment (windy-worm) [inline EngineBay.Authentication/JwtBearerAuth/CurrentIdentityFromJwt.cs:16]: log a ticket

**Review by windy-worm — APPROVED** [review]

**Pull request merged**